platform/config/go-links.yaml

```
@@ -0,0 +1,4 @@
+# Use via: go.amp.dev/samples
+/samples: /documentation/examples/
+/docs: /documentation/
+
```

I remember Matt asked for:

```suggestion
/websites: /about/websites
/stories: /about/stories
/ads: /about/ads
/email: /about/email
```